src/nxdoc/nuxeo-server/core-server/automation/automation-scripting.md

```diff
@@ -432,22 +426,31 @@ function run(input, params) {
   // Then all comparaisons can be made
   var compare = (nowDate > releaseDate);
   if (compare) {
-    WebUI.AddInfoMessage(
-      input, {
-        'message': 'Now less than release date'
-      }
-    );
+    Console.log("Now less than release date");
   }
   else {
-    WebUI.AddInfoMessage(
-      input, {
-        'message': 'Now greater than release date'
+    Console.log("Now greater than release date");
       }
     );
   }
 }
 ```
 
+### Event Context
+
+It is possible to access to the event context. This can be really usefull when trying to access some repository information before the document is created: Typically, you can't access parent properties on the `About to create` event without `ctx.Event.getProperty`:
+
+
+```js
+  function run(input, params) {
+    /* Use parentPath for Empty document created event, and parentRef for About to create event */
+    var parentPath = ctx.Event.getProperty("parentPath");
+    var parentDoc = Repository.GetDocument(null, {"value": parentPath});
+    ...
```

Comment:
Formatting:
- `function` should start at the beginning of the line
- you should close the open bracket